.github/workflows/psalm.yml

```diff
@@ -19,79 +19,31 @@ jobs:
 
       - name: Checkout PR
         uses: actions/checkout@v2
-        with:
-          path: pr
-
-      - name: Checkout base
-        uses: actions/checkout@v2
-        with:
-          ref: ${{ github.base_ref }}
-          path: base
 
       - name: Configure composer
         run: |
-            cd base
             COMPOSER_HOME="$(composer config home)"
             ([ -d "$COMPOSER_HOME" ] || mkdir "$COMPOSER_HOME") && cp .github/composer-config.json "$COMPOSER_HOME/config.json"
             echo "COMPOSER_ROOT_VERSION=$(grep -m1 SYMFONY_VERSION .travis.yml | grep -o '[0-9.x]*').x-dev" >> $GITHUB_ENV
 
-      - name: Determine composer cache directory
-        id: composer-cache
-        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
-
-      - name: Cache composer dependencies
-        uses: actions/cache@v2
-        with:
-          path: ${{ steps.composer-cache.outputs.dir }}
-          key: composer-${{ github.base_ref }}
-          restore-keys: composer-
-
       - name: Install Psalm
         run: |
-          composer require psalm/phar
-          cp ./vendor/bin/psalm.phar base/psalm.phar
-          cp ./vendor/bin/psalm.phar pr/psalm.phar
-
-      - name: Install dependencies for base
-        run: |
-            cd base
             echo "::group::modify composer.json"
-            composer remove symfony/phpunit-bridge --no-interaction --no-update
-            composer require --no-update phpunit/phpunit php-http/discovery psr/event-dispatcher
+            composer remove --no-update --no-interaction symfony/phpunit-bridge
+            composer require --no-update psalm/phar phpunit/phpunit php-http/discovery psr/event-dispatcher
```

Comment:
is all this a workaround for something?
does downloading the phar directly help?
https://github.com/vimeo/psalm/blob/master/docs/running_psalm/installation.md#using-the-phar

Reply:
this is for installing some optional deps that psalm will complain about otherwise